modules/nextflow/src/main/groovy/nextflow/k8s/K8sConfig.groovy

```diff
@@ -55,11 +55,14 @@ class K8sConfig implements Map<String,Object> {
             this.podOptions.volumeClaims.add(new PodVolumeClaim(name, mount, subPath))
         }
 
-        // -- shortcut to pod image pull-policy
-        if( target.pullPolicy )
+        // -- shortcut to pod image pull policy
+        if( target.pullPolicy ) {
+            log.warn "The `pullPolicy` option is deprecated -- Use `imagePullPolicy` instead."
```

Comment:
In principle, only the one in `PodOption` should be enough. It does not?

Reply:
Apparently it is both a pod option and a `k8s` config option, so I deprecated both.